src/js/extensions/infragistics.ui.combo.knockout-extensions.js

```diff
@@ -16,17 +16,16 @@
 	if (typeof define === "function" && define.amd) {
 		define( [
             "jquery",
-            "jquery-ui",
             "knockout",
             "../modules/infragistics.util",
             "../modules/infragistics.datasource",
             "../modules/infragistics.ui.combo"
```

Comment:
I believe that the combo already has dependency on the datasource and the util. Could you check that also for tree and editors?